test/spectests/spectests.cpp

```diff
@@ -51,11 +51,23 @@ const auto spectest_module = fizzy::parse(spectest_bin);
 const std::string spectest_name = "spectest";
 
 template <typename T>
-uint64_t json_to_value(const json& v)
+fizzy::Value json_to_value(const json& v)
 {
     return static_cast<std::make_unsigned_t<T>>(std::stoull(v.get<std::string>()));
 }
 
+template <>
+fizzy::Value json_to_value<float>(const json& v)
+{
+    return std::stoull(v.get<std::string>());
```

Comment:
Why using "to unsigned integer" for floating point values?

Reply:
`wast2json` puts them as 64-bit integers to json